search/processor.cpp

```diff
@@ -591,10 +598,15 @@ void Processor::InitGeocoderParams(Geocoder::Params & params)
   InitParams(params);
   params.m_mode = m_mode;
   params.m_pivot = GetPivotRect();
-  params.m_accuratePivotCenter = GetPivotPoint();
   m_geocoder.SetParams(params);
 }
 
+void Processor::InitPreRankerParams(PreRanker::Params & params)
+{
+  params.m_accuratePivotCenter = GetPivotPoint();
+  m_preRanker.SetParams(params);
```

Comment:
The method name is disguising, as it not only inits PreRanker params, but also inits PreRanker with them.